root/static/scripts/guess-case/modes.js

```diff
@@ -8,7 +8,7 @@ const {assign} = require('lodash');
 
 const {l} = require('../common/i18n');
 const getBooleanCookie = require('../common/utility/getBooleanCookie');
-const {isPrepBracketWord, isPrepBracketSingleWord} = require('./utils');
+const {isPrepBracketWord, isPrepBracketSingleWord, turkicUpperCase, turkicLowerCase} = require('./utils');
```

Comment:
I proposed it at purpose in https://github.com/metabrainz/musicbrainz-server/pull/410#discussion_r93604082, since dotless/dotted I are also used by _Azerbaijanis and Khazaks_ (when using latin script), see the final note to [MBS-1319](https://tickets.metabrainz.org/browse/MBS-1319).